packages/meditrak-server/src/apiV2/import/importSurveyResponses/importSurveyResponses.js

```diff
@@ -313,9 +313,13 @@ async function getNewDataTimeIfRequired(models, surveyResponseId, newDataTime) {
 function getDateStringForColumn(sheet, columnIndex) {
   const dateString = getInfoForColumn(sheet, columnIndex, 'Date');
   const isInExportFormat = moment(dateString, EXPORT_DATE_FORMAT, true).isValid();
-  return isInExportFormat
-    ? stripTimezoneFromDate(moment(dateString, EXPORT_DATE_FORMAT).toDate())
-    : stripTimezoneFromDate(moment(dateString).toDate());
+  const date = isInExportFormat
+    ? moment(dateString, EXPORT_DATE_FORMAT).toDate()
+    : moment(dateString).toDate();
+  if (isNaN(date.getTime())) {
+    throw new ImportValidationError(`Invalid date ${dateString}`);
+  }
+  return stripTimezoneFromDate(date);
```

Comment:
@EMcQ-BES now tackling 3 bugs in one PR. Saves some overhead, but means you need to review yet again